flizpaysdk/src/main/kotlin/flizpay2/flizpaysdk/lib/WebViewService.kt

```diff
@@ -27,18 +41,49 @@ class WebViewService : AppCompatActivity() {
             runOnUiThread { finish() } // Close activity when triggered
         }
 
+        webView.webViewClient = object : WebViewClient() {
+
+            override fun shouldOverrideUrlLoading(
+                view: WebView?,
+                request: WebResourceRequest?
+            ): Boolean {
+                val url = request?.url ?: return false
+                val host = url.host ?: return false
+                
+                if (url.scheme.equals("androiddemo", ignoreCase = true)) {
```

Comment:
```suggestion
                if (url.scheme.equals("flizpaywebview", ignoreCase = true)) {
```

Reply:
We probably should move this to constants so we can mock and change properly during local development